descwl_coadd/tests/test_coadd_correct.py

```diff
@@ -78,7 +78,7 @@ def test_coadd_image_correct(crazy_wcs, crazy_obj):
     world_origin = galsim.CelestialCoord(0 * galsim.degrees, 0 * galsim.degrees)
 
     aff = galsim.PixelScale(scale).affine()
-    aff = aff.withOrigin(
+    aff = aff.shiftOrigin(
```

Comment:
Why is `shiftOrigin` more appropriate than `withOrigin` here?

Reply:
`withOrigin` is basically a call to `shiftOrigin` with a deprecation warning:

https://github.com/GalSim-developers/GalSim/blob/f6499baeadf433a34f98f5c918b2580e6d28bf7b/galsim/wcs.py#L699-L702